packages/table/src/regions.ts

```diff
@@ -528,6 +528,20 @@ export class Regions {
         return regionGroups;
     }
 
+    public static regionArraysEqual(regionsA: IRegion[], regionsB: IRegion[]) {
```

Comment:
i might call this `DeepEqual` to clarify that it's much more than a `===` check.